### PR TITLE
chore: plagiarism correction in index.js

### PR DESCRIPTION
**Pull request by agathambrose**
I made a correction to the subheading where what a dungeon is, was defined.
If you run a search on "dungeon" you will see that definition will be the first thing you see on google. It was written in Wikipedia or follow this link to confirm https://en.wikipedia.org/wiki/Dungeon#:~:text=A%20dungeon%20is%20a%20room,more%20to%20the%20Renaissance%20period.

In other words, to avoid plagiarism, I corrected that phrase. 

Lovely game by the way!

### CODE REVIEW

**Comment by ming-tsai** [comment]
Hi @agathambrose,
Could you remove the file change package.json and package-lock.json

**Comment by agathambrose** [comment]
Hi @ming-tsai sure I can! 
Will do that now.

**Comment by agathambrose** [comment]
@ming-tsai Done!

File: package.json [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-    "react-device-detect": "^1.14.0",
-    "react-dom": "^16.8.6",
-    "react-helmet": "^6.1.0",
-    "react-scripts": "^4.0.3",
-    "react-typography": "^0.16.19",
-    "typography": "^0.16.19",
-    "typography-plugin-code": "^0.16.19",
-    "typography-theme-github": "^0.16.19"
-  }
-}
```
Comment (ming-tsai) [inline package.json:41]: Please do no delete this file?

Comment (ming-tsai) [reply package.json:41]:
```json
{
  "name": "markdown-dungeon",
  "version": "1.0.0",
  "repository": "https://github.com/MakeContributions/markdown-dungeon.git",
  "author": "MakeContributions <tsaibot@outlook.com>",
  "license": "MIT",
  "scripts": {
    "start": "gatsby develop",
    "build": "gatsby build",
    "serve": "gatsby serve"
  },
  "dependencies": {
    "@material-ui/core": "^4.11.3",
    "@material-ui/icons": "^4.11.2",
    "gatsby": "^3.2.0",
    "gatsby-image": "^2.4.20",
    "gatsby-plugin-catch-links": "^2.3.14",
    "gatsby-plugin-manifest": "^3.2.0",
    "gatsby-plugin-offline": "^3.2.30",
    "gatsby-plugin-react-helmet": "^3.3.12",
    "gatsby-plugin-sharp": "^2.6.38",
    "gatsby-plugin-typography": "^2.5.12",
    "gatsby-remark-autolink-headers": "^2.3.14",
    "gatsby-remark-prismjs": "^3.5.15",
    "gatsby-source-filesystem": "^2.3.32",
    "gatsby-transformer-remark": "^2.8.37",
    "gatsby-transformer-sharp": "^2.5.16",
    "prism-themes": "^1.4.1",
    "prismjs": "^1.23.0",
    "prop-types": "^15.7.2",
    "react": "^16.8.6",
    "react-device-detect": "^1.14.0",
    "react-dom": "^16.8.6",
    "react-helmet": "^6.1.0",
    "react-scripts": "^4.0.3",
    "react-typography": "^0.16.19",
    "typography": "^0.16.19",
    "typography-plugin-code": "^0.16.19",
    "typography-theme-github": "^0.16.19"
  }
}
```

Comment (agathambrose) [reply package.json]: Package.json restored!

**Review by ming-tsai — APPROVED** [review]
Approved

**Comment by ming-tsai** [comment]
Hi @agathambrose,
Nice work, and welcome to our team
> The invitation [link](https://github.com/MakeContributions)

**Pull request merged**